doc/architecture_decisions/ADR-000-adr-template.md

```diff
@@ -0,0 +1,34 @@
+# Title: ADR-000: Architectural Decision Record Template
+
+## Status: Accepted|Rejected|Proposed
+
+## Context
+
+What is the motivation for this change?
+
+## Decision
+
+Detail content of what we are designed.
+
+Detail on other possible approaches and why we are not using them.
+
+## Consequences
+
+### Better
+
+What's the better outcome could this design brings.
+
+### Worse
+
+What's the worse outcome could this design brings, please also explain
+why it is not veto this ADR.
```

Comment:
Do you mean "please explain why this ADR is not vetoed", honestly, it is better to rephrase it a bit for better clarity.